hosts/signed-int16/home/reisen/home.nix

```diff
@@ -7,15 +7,60 @@
   ...
 }: {
   imports =
-    [
+    [ 
       (path + /modules/shared/home/reisen/programs/dev/vim.nix)
     ]
     ++ (import (path + /modules/shared/home/reisen/programs/utils));
   home = {
     username = "reisen";
     packages = with pkgs; [
+      # Audio
+      tenacity
+      pavucontrol
+      qpwgraph
+      
+      # BROWSERS
+      ungoogled-chromium
+      firefox
+      librewolf
+
+      # Graphics
+      krita
+      gimp
+      inkscape
+      kdenlive
+
+      # Media
+      quodlibet
+      vlc
+
+      # Dev
+      lazygit
+      vscodium-fhs
+
+      # Gaming
+      steam
+      steam-tui
+      steam-run
```

Comment:
Steam is not declared like this, it won't matter if it's declared system wide or in specific userspace, you will face issues with libraries and compatibility especially when it comes to native games
```suggestion
```
And use nixos steam [options](https://nixos.wiki/wiki/Steam)

Reply:
> it won't matter if it's declared system wide or in specific userspace, you will face issues with libraries and compatibility especially when it comes to native games
what do you suggest instead of declaring it in my config? do you think the steam flatpak is a better choice?

Reply:
For a reference you can check `modules/unsigned-int32/programs/steam.nix`
You already have a `programs.steam.enable = true;` if you're not planning on running steam remote play it will be enough, you will still have `steam`, `steam-run`, ?`steam-tui` packages respectively

Reply:
stole your steam config :3

Reply:
legit